### ISSUE TITLE
Issue on multiple pages but not sure which. "Choose" should be changed to "Select"

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1444]
## Description of issue
We should use choose when referring to a customer decision that isn’t UI-related. Use select for anything a user clicks or taps after.

Describe what is failing
I know I've used "choose" at times because I thought it was more plain language. I'm not sure where I used it. 

## Potential fix

"Choose" could be changed to "select" throughout Notify.

## Resources

- UX Content Collective course: Accessibility for UX writers and designers links to this Intuit resource: https://contentdesign.intuit.com/product-and-ui/writing-about-ui/

## Acceptance criteria

- [x] Find occurence of issue and document in inventory doc.
- [x] Update content for those occurences in EN and FR in content doc called [Ally sprint -Notify choose v select update](https://docs.google.com/document/d/1W0DiQcN_ncTWeNaDc71xC90l09Pc-EVpWwsBvf-LjSk/edit#heading=h.g9do9paa95d0)
- [ ] Content designer updates GC Articles based on [Ally sprint -Notify choose v select update](https://docs.google.com/document/d/1W0DiQcN_ncTWeNaDc71xC90l09Pc-EVpWwsBvf-LjSk/edit#heading=h.g9do9paa95d0)
- [x] Developer updates code based on [Ally sprint -Notify choose v select update](https://docs.google.com/document/d/1W0DiQcN_ncTWeNaDc71xC90l09Pc-EVpWwsBvf-LjSk/edit#heading=h.g9do9paa95d0) **Note: Developer updates start on page 13 of this doc and are indicated with the comment function**
PR: https://github.com/cds-snc/notification-admin/pull/1829

**Comment by YedidaZalik** [comment]
Auditing here https://docs.google.com/spreadsheets/d/1Eo6Yj9hMl5hjoBc_FSP9YPb9Cr-o7VroENyWC6HrZEI/edit#gid=0
Checking against IA https://www.figma.com/file/qbJeSvvVrPp9OsULCtozRI/Exploring-and-planning-%7C-Explorer-et-planifier?type=design&node-id=1127-2&mode=design&t=qySelx6uKaPSweZi-0

**Comment by YedidaZalik** [comment]
Content in google doc [Notify choose v select content update](https://docs.google.com/document/d/1W0DiQcN_ncTWeNaDc71xC90l09Pc-EVpWwsBvf-LjSk/edit#heading=h.g9do9paa95d0)

**Comment by yaelberger-commits** [comment]
Mostly done, Yedida scheduling a meeting with Phil to review

**Comment by YedidaZalik** [comment]
To review with Phil and MSB next week

**Comment by YedidaZalik** [comment]
Meeting scheduled

**Comment by YedidaZalik** [comment]
Marie-Sophie consulting with Anik about rationale to use  "symbol" instead of "logo" on Forms 
Considering consistency between terms on platform products
Anik looking for their documentation about that

**Comment by YedidaZalik** [comment]
French is ready 
EN and FR both in this doc https://docs.google.com/document/d/1W0DiQcN_ncTWeNaDc71xC90l09Pc-EVpWwsBvf-LjSk/edit
Yedida to add to GC Articles and ask devs how best to proceed product screens

**Comment by YedidaZalik** [comment]
From Melissa  ([thread](https://gcdigital.slack.com/archives/G010A3CDV5X/p1713879846826039)): I can confirmed, after consulting Sam and Nourhan, that Notify's use of the term logo makes sense for our context. My understanding is that this is consistent with how Forms uses the term (see [link](https://articles.alpha.canada.ca/forms-formulaires/request-alternate-branding/)).
It's also my understanding that design system is a bit different as neither GC Design System or the [Canada.ca](http://canada.ca/) design system have components specifically for depts to add their own logos. This makes sense as these cases would be exceptions. Instead the [Government of Canada signature](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/design-standard/colour-design-standard-fip.html) and the [Canada wordmark](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/design-standard/colour-design-standard-fip.html) are required as headers and footers, respectively.

**Comment by YedidaZalik** [comment]
Yedida to start inputting the new content today

**Comment by YedidaZalik** [comment]
Met with Jumana and Phil about UI screens. Phil added in AC and I updated card description based on meeting.

I updated the GC Articles pages and cleared the cache. Checked pages again and they look good.
